signer/tests/integration/transaction_signer.rs

```diff
@@ -275,3 +278,84 @@ async fn get_signer_public_keys_and_aggregate_key_falls_back() {
 
     testing::storage::drop_db(db).await;
 }
+
+/// Test that [`TxSignerEventLoop::handle_pending_deposit_request`] does
+/// not error when attempting to check the scriptPubKeys of any of the
+/// inputs of a deposit.
+#[cfg_attr(not(feature = "integration-tests"), ignore)]
+#[tokio::test]
+async fn handle_pending_deposit_request_address_script_pub_key() {
+    let db_num = DATABASE_NUM.fetch_add(1, Ordering::SeqCst);
+    let db = testing::storage::new_test_database(db_num, true).await;
+
+    let mut rng = rand::rngs::StdRng::seed_from_u64(51);
+
+    let ctx = TestContext::builder()
+        .with_storage(db.clone())
+        .with_mocked_clients()
+        .build();
+
+    let (rpc, faucet) = sbtc::testing::regtest::initialize_blockchain();
+
+    // This confirms a deposit transaction, and has a nice helper function
+    // for storing a real deposit.
+    let setup = TestSweepSetup::new_setup(rpc, faucet, 10000, &mut rng);
+
+    // Let's get the blockchain data into the database.
+    let chain_tip: BitcoinBlockHash = setup.sweep_block_hash.into();
+    backfill_bitcoin_blocks(&db, rpc, &chain_tip).await;
+
+    // We need to store the deposit request because of the foreign key
+    // constraint on the deposit_signers table.
+    setup.store_deposit_request(&db).await;
+
+    // In order to fetch the deposit request that we just store, we need to
+    // store the deposit transaction.
+    setup.store_deposit_tx(&db).await;
+
+    let mut requests = db
+        .get_pending_deposit_requests(&chain_tip, 100)
+        .await
+        .unwrap();
+    // There should only be the one deposit request that we just fetched.
+    assert_eq!(requests.len(), 1);
+    let request = requests.pop().unwrap();
+
+    let network = InMemoryNetwork::new();
+    let mut coord = TxSignerEventLoop {
```

Comment:
nit:
```suggestion
    let mut signer = TxSignerEventLoop {
```

Reply:
Done in https://github.com/stacks-network/sbtc/pull/675/commits/0902b2a9b7a79745473e7b90405ef5e4860113d7.